buildpipeline/Core-Setup-Windows-BT.json

```diff
@@ -60,7 +60,62 @@
       "enabled": true,
       "continueOnError": false,
       "alwaysRun": false,
-      "displayName": "Build",
+      "displayName": "Install Signing Plugin",
```

Comment:
although this makes it hard to review because the build defs got switched around.  Oh well.